crates/c_api/macro/lib.rs

```diff
@@ -27,6 +27,7 @@ pub fn declare_own(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
     (quote! {
         #[doc = #docs]
         #[no_mangle]
+        #[wasmi_c_api_macros::prefix_symbol]
```

Comment:
Please use `#[cfg_attr(feature = "prefix-symbols", prefix_symbol)]` in all usage sites as I recommended earlier. The reason is that this way we avoid running the proc. macro numerous times which is exceptionally expensive when not needed.

Reply:
With this `cfg_attr` guard the `prefix_symbol` proc. macro should expand to a non-`cfg_attr` but a normal `cfg` attribute.

Reply:
You're totally right.